DependencyInjection/Compiler/BuilderPass.php

```diff
@@ -66,14 +68,23 @@ private function defineKeyChainRepository(ContainerBuilder $container, array $co
         return $this;
     }
 
+    private function defineNonceRepository(ContainerBuilder $container, array $configuration): self
```

Comment:
Not sure you need the configuration here :smile:

Reply:
not yet, true :)
will remove for now,. it's better